plugins/carla/cmake/jackbridge.cmake

```diff
@@ -0,0 +1,23 @@
+# base config
+set(carla_jackbridge_basedir carla/source/jackbridge)
+set(carla_jackbridge_extra_libs ${carla_pthread_libs})
+
+if(NOT
```

Comment:
Possibly easier to check specifically for `OS_LINUX` if this is just necessary on Linux.